services/core-web/src/components/Forms/incidents/IncidentForm.tsx

```diff
@@ -142,794 +143,820 @@ const retrieveInitialReportDynamicValidation = (childProps) => {
-                format={null}
-                placeholder="Search for responsible inspector..."
-                data={childProps.inspectorOptions}
-                disabled={!isEditMode}
-              />
-            </Form.Item>
-          </Col>
-          <Col md={12} xs={24}>
-            <Form.Item label="Was there a follow-up inspection?">
-              <Field
-                id="followup_inspection"
-                name="followup_inspection"
-                component={renderConfig.RADIO}
-                disabled={!isEditMode}
+              <Col xs={24} md={6}>
+                {!isClosed && childProps.isEditMode && (
+                  <Form.Item>
+                    <Field
+                      id="status_code"
+                      name="status_code"
+                      label=""
+                      placeholder="Action"
+                      component={renderConfig.SELECT}
+                      validate={[required]}
+                      data={childProps.dropdownIncidentStatusCodeOptions}
+                    />
+                  </Form.Item>
+                )}
+                {!childProps.pristine && !isClosed && (
+                  <div className="right center-mobile">
+                    <Button
+                      className="full-mobile"
+                      type="primary"
+                      htmlType="submit"
+                      disabled={selectedStatusCode === "UNR" && !responsibleInspector}
+                    >
+                      Update Status
+                    </Button>
+                  </div>
+                )}
+              </Col>
+            </Row>
+          }
+          type={!isClosed ? "warning" : "info"}
+          showIcon
+          style={{
+            backgroundColor: isClosed ? "#FFFFFF" : "",
+            border: isClosed ? "1.5px solid #5e46a1" : "",
+          }}
+          className={isClosed ? "ant-alert-info ant-alert-info-custom-core-color-icon" : null}
+        />
+      </Col>
+    ) : null;
+  };
+
+  const renderEditSaveControls = (childProps, isEditMode, isNewIncident) => (
+    <div className="right center-mobile violet">
+      {isEditMode && (
+        <Button
+          id="mine-incident-submit"
+          className="full-mobile right"
+          type="primary"
+          htmlType="submit"
+          loading={childProps.submitting}
+          disabled={childProps.submitting}
+        >
+          {isNewIncident ? "Create Incident" : "Save Changes"}
+        </Button>
+      )}
+    </div>
+  );
+
+  const renderInternalDocumentsComments = (childProps, isEditMode, handlers, parentHandlers) => {
+    const incidentCreated = Boolean(childProps.formValues?.mine_incident_guid);
+    const internalMinistryDocuments = childProps.documents.filter(
+      (doc) => doc.mine_incident_document_type_code === "INM"
+    );
+    const dispatch = useDispatch();
+
+    const handleRemoveDocument = async (event, documentGuid: string) => {
```

Comment:
This is the one change in the actual code.  Handling the deletion of documents here.  And then just calling the callback for cleanup.